MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -252,7 +252,13 @@ private async Task SavePeopleMetadataAsync(List<PersonInfo> people, LibraryOptio
 
                     if (!string.IsNullOrWhiteSpace(person.ImageUrl) && !personEntity.HasImage(ImageType.Primary))
                     {
-                        await AddPersonImageAsync(personEntity, libraryOptions, person.ImageUrl, cancellationToken).ConfigureAwait(false);
+                        personEntity.SetImage(
```

Comment:
Is this approach right btw? You pointed me at a single line, but did you want me to remove that function completely? @cvium 

This couldn't be in the async function that you told me to remove because it's a ``void``, that's why I moved it here.

Reply:
Yes